fiddler/tox.ini

```diff
@@ -20,5 +20,6 @@ deps =
 passenv =
     DOCKER*
     COMPOSE*
+    FIDDLER_API_KEY
```

Comment:
Passes the local `FIDDLER_API_KEY` environment variable to the testing environment

Reply:
Looks good.